src/Core/Client/Client.php

```diff
@@ -766,7 +767,13 @@ public function removePlugin($plugin): ClientInterface
     public function createRequest(QueryInterface $query): Request
     {
         $event = new PreCreateRequestEvent($query);
-        $this->eventDispatcher->dispatch(Events::PRE_CREATE_REQUEST, $event);
+        if (Kernel::VERSION_ID >= 40300) {
+            // Support for symfony listeners which are using the old event name.
+            $this->eventDispatcher->dispatch($event, Events::PRE_CREATE_REQUEST);
+            $this->eventDispatcher->dispatch($event);
```

Comment:
this fires the event twice without any form of sanity check so will break event dispatchers with one argument